Package.swift

```diff
@@ -79,7 +79,9 @@ let package = Package(
         // Test utility library
         .target(
             name: "SwiftDocCTestUtilities",
-            dependencies: []),
+            dependencies: [
+                "SymbolKit"
```

Comment:
Mostly out of curiosity: what functionality of the utilities target is dealing with SymbolKit directly? My understanding is that the utilities target is mainly meant to cover the command-line interface and its related functionality but not any core logic.

Reply:
I factored out `makeSymbolGraph(moduleName:symbols:relationships:)` into `XCTestCase` as it was used in more than one test file. See [Sources/SwiftDocCTestUtilities/SymbolGraphCreation.swift](https://github.com/apple/swift-docc/blob/4b52aefb65a739cdf16d49300b7d592d7e8882a5/Sources/SwiftDocCTestUtilities/SymbolGraphCreation.swift)